### PR TITLE
Merge DemoBench into Corda.

### PR DESCRIPTION
**Pull request by chrisr3**
The DemoBench tool is intended to become a sales and educational tool for Corda. It is a standalone desktop app that comes bundled with an appropriate JVM, and which runs nodes in a local network. EXE, DMG and RPM installers are created for Windows, Mac and Linux distributions using javapackager.

### CODE REVIEW

File: tools/demobench/build.gradle [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                include '**/*.wsf'
+                include '**/*.manifest'
+            }
+            filter {
+                line -> line.replaceAll('@pkg_version@', pkg_version)
+            }
+            into "$pkg_source/package"
+        }
+
+        ant.taskdef(
```
Comment (Clintonio) [inline tools/demobench/build.gradle]: Is there not a more gradle native way to do this than via Ant?

Comment (chrisr3) [inline tools/demobench/build.gradle]: 'Fraid not. Javapackager is via Ant task or command line.

Comment (Clintonio) [inline tools/demobench/build.gradle]: Yeah, that's reasonable. I was just wondering if there was a gradle plugin for it. [This one](https://github.com/FibreFoX/javafx-gradle-plugin) looks relevant, but if you think it's not the right way I'll be done with the review.

Comment (chrisr3) [inline tools/demobench/build.gradle]: That Gradle plugin is just a wrapper around the Ant task, so all I think we'd be doing is adding complexity.

**Pull request merged**